test/blackbox-tests/test-cases/alias-module.t

```diff
@@ -16,3 +16,5 @@ Test alias module for wrapped libraries:
   
   (** @canonical Foo.Bar *)
   module Bar = Foo__Bar
+  
+  module Foo__ = struct let shadowed = () end
```

Comment:
is `shadowed` useful to something in particular or could `struct end` be enough?

Reply:
I suppose I can also just add an alert that will be triggered when the binding will be used.

Reply:
Good idea, that detects `module X = Foo__`. But the correct syntax is

```
module Foo__ = struct end
[@@deprecated "this module is shadowed"]
```